src/orion/algo/space.py

```diff
@@ -568,6 +585,11 @@ def __init__(self, name, categories, **kwargs):
                                                   self._probs))
         super(Categorical, self).__init__(name, prior, **kwargs)
 
+    @property
+    def capacity(self):
+        """Return the capacity size of all the possible values from Categorical `Dimension`"""
+        return len(self.categories)
```

Comment:
This should depend on shape as well, isn't?

Reply:
I did not realize that Categorical will also support shape. I will update this.